src/Runtime/Fork/ForkRuntime.php

```diff
@@ -66,6 +66,7 @@ public function defer(Closure $callback): Future
         }
 
         if ($pid === 0) {
+            // @codeCoverageIgnoreStart
```

Comment:
I believe xdebugs coverage driver does not detech coverage on a child process. This is why I added the ignore in this position